ppd/array-private.h

```diff
@@ -33,6 +33,8 @@ extern "C" {
 extern int		_ppdArrayAddStrings(cups_array_t *a, const char *s,
 			                     char delim);
 extern cups_array_t	*_ppdArrayNewStrings(const char *s, char delim);
+extern int      ppdArrayAdd(cups_array_t *a, const char   *s);
+extern cups_array_t *ppdArrayNew(const char *s);
```

Comment:
Please rename these (library-internal) functions to start with `_ppd` (add an underscore `_` to the beginning of the name).

Reply:
Also, I think we do not need to create these two library-internal API functions. what they do is trivial.

There is only one call of `ppdArrayNew()`. You can directly put something like
```
if ((output = cupsArrayNew3(NULL, NULL, NULL, 0,
                            (cups_acopy_func_t)_ppdStrAlloc,
                            (cups_afree_func_t)_ppdStrFree)) == NULL)
{
  if (log) log(ld, CF_LOGLEVEL_ERROR,
               "ppdTest: Could not allocate memory."); 
  return(NULL);
}
```
there.

Each call of `ppdArrayAdd(output, "...")` you can replace by
```
cupsArrayAdd(output, (void *)"...");
```
and you get the same result.